docs/import/create_score_asset.js

```diff
@@ -22,16 +22,20 @@ function countDamageAndBuildings(feature, damage, buildings) {
   const geoId = feature.get(geoidTag);
   const totalBuildings = ee.Algorithms.If(
       buildings.contains(geoId), buildings.get(geoId), ee.Number(0));
+  const rawIncomeValue = ee.String(feature.get(incomeTag));
+  const income = ee.Algorithms.If(
+      ee.Algorithms.IsEqual(rawIncomeValue, ee.String('250,000+')),
+      ee.Number(250000),
+      ee.Algorithms.If(
+          ee.Algorithms.IsEqual(rawIncomeValue, ee.String('-')), null,
+          ee.Number.parse(rawIncomeValue)));
```

Comment:
should we just filter out all non-numeric values? Are we sure these are only ones? We could do for svi too as a safety measure (other values are known numbers). For example, here is functionality to strip out all non-numeric values:

```
// Permissive number regexp: matches optional whitespace followed by optional
// minus sign followed by 0 or more digits followed by optional period with 0 or
// more digits followed by optional whitespace. Note that it matches an empty
// string. Besides empty string, corresponds to valid inputs to ee.Number.parse.
const numberRegexp = '^ *-?[0-9]*(\.[0-9]*) *$';

function convertToNumber(value) {
  value = ee.String(value);
  return ee.Algorithms.If(
      value.length().and(value.match(numberRegexp).length()),
      ee.Number.parse(value),
      null);
}

```

Reply:
Done. With updated comments.